src/app/pages/MarginTradePage/utils/marginUtils.ts

```diff
@@ -0,0 +1,70 @@
+import { weiTo18 } from 'utils/blockchain/math-helpers';
+import { bignumber } from 'mathjs';
+import type { OpenLoanType } from 'types/active-loan';
+import { TradingPosition } from 'types/trading-position';
+import imgArrowUp from 'assets/images/trend-arrow-up.svg';
+import imgArrowDown from 'assets/images/trend-arrow-down.svg';
+import { TradePriceChange } from '../types';
+
+export function getEntryPrice(item: OpenLoanType, position: TradingPosition) {
```

Comment:
Rewrite all of the functions here to `const`.

Reply:
done